system-tests/tests/smoke/cre/por_test.go

```diff
@@ -386,14 +411,18 @@ func setupPoRTestEnvironment(
 	containerPath, pathErr := capabilities.DefaultContainerDirectory(in.Infra.InfraType)
 	require.NoError(t, pathErr, "failed to get default container directory")
 	var cronBinaryPathInTheContainer string
+	var readContractPathInTheContainer string
 	if in.WorkflowConfig.DependenciesConfig.CronCapabilityBinaryPath != "" {
 		// where cron binary is located in the container
 		cronBinaryPathInTheContainer = filepath.Join(containerPath, filepath.Base(in.WorkflowConfig.DependenciesConfig.CronCapabilityBinaryPath))
+		readContractPathInTheContainer = filepath.Join(containerPath, filepath.Base(in.WorkflowConfig.DependenciesConfig.ReadContractCapabilityBinaryPath))
 		// where cron binary is located on the host
 		customBinariesPaths[keystonetypes.CronCapability] = in.WorkflowConfig.DependenciesConfig.CronCapabilityBinaryPath
+		customBinariesPaths[keystonetypes.ReadContractCapability] = in.WorkflowConfig.DependenciesConfig.ReadContractCapabilityBinaryPath
 	} else {
 		// assume that if cron binary is already in the image it is in the default location and has default name
 		cronBinaryPathInTheContainer = filepath.Join(containerPath, "cron")
+		readContractPathInTheContainer = filepath.Join(containerPath, keystonetypes.ReadContractCapability)
```

Comment:
this should rather say:
```
readContractPathInTheContainer = filepath.Join(containerPath, "amd64_readcontract")
```

because the second argument is the default binary name